src/main/java/com/sap/sgs/phosphor/fosstars/model/score/oss/CommunityCommitmentScore.java

```diff
@@ -61,4 +69,43 @@ public ScoreValue calculate(Value... values) {
         Score.adjust(scorePoints),
         Confidence.make(hasResponsibleCompany, isApacheProject, isEclipseProject));
   }
+
+  /**
+   * This class implements a verification procedure for {@link CommunityCommitmentScore}.
+   * The class loads test vectors, and provides methods to verify a {@link CommunityCommitmentScore}
+   * against those test vectors.
+   */
+  public static class Verification extends ScoreVerification {
+
+    /**
+     * A name of a resource which contains the test vectors.
+     */
+    private static final String TEST_VECTORS_CSV = "CommunityCommitmentScoreTestVectors.csv";
+
+    /**
+     * Initializes a {@link Verification}
+     * for a {@link CommunityCommitmentScore}.
+     *
+     * @param score A score to be verified.
+     * @param vectors A list of test vectors.
+     */
+    public Verification(CommunityCommitmentScore score, List<TestVector> vectors) {
+      super(score, vectors);
+    }
+
+    /**
+     * Creates an instance of {@link Verification} for a specified score. The method loads test
+     * vectors from a default resource.
+     *
+     * @param score The score to be verified.
+     * @return An instance of {@link CommunityCommitmentScore}.
+     */
+    static Verification createFor(CommunityCommitmentScore score) throws IOException {
```

Comment:
Hmm, just a question. If this method is needed by classes outside this class, shouldn't this be public static?

Reply:
It's currently used only by classes in the same package. That's why it's package-private. I would like to restrict access as much as I can.